configs/components/_base-ruby.rb

```diff
@@ -140,11 +116,27 @@
   #                   -Sean P. McDonald 07/01/2016
   lib_type = platform.architecture == "x64" ? "seh" : "sjlj"
 
+  if platform.architecture == "x64"
+    gcc_postfix = 'seh'
```

Comment:
Super bikesheddy nitpick: would `suffix` be better?

Reply:
ha, yeah.